### ISSUE TITLE
Implement SignalR

### ISSUE THREAD

**Issue by PatrykPorabik** [grandnode/grandnode#401]
For example: 
- "Realtime" alerts when participating in an auction to let you know you've been outbid.  
- Notices about status changes with orders  (i.e. back orders, shipped, etc).  
- Live notifications about specials for those currently browsing the page, etc. 
- If the application is going to be restarted for whatever reason, live notifications to advise current guests. 
- Notifications to advise of price changes on products either previously viewed or in wish list/shopping cart.
- Live chat (might be easier to offload this one to other easy free services though)

**Comment by KrzysztofPajak** [comment]
Postponed - we waiting for blazor

**Comment by Leonardo-Ferreira** [comment]
My 2 cents are: These are all nice features but I think we should implement this with a pooling strategy first. All the cases are perfect examples of in-memory cache (no need to be distributed cache). In my point of view, apart from the chat, none of these need to be "real-time", if any of them are delivered with a 10sec to 5 min delay, no harm would incur...

**Comment by PatrykPorabik** [comment]
You're right. It was just an example coming from our community forum, maybe we will focus on it after 4.60 release. But we can start collecting the ideas for future scope of the feature.
